packages/simtask/package.json

```
@@ -20,9 +20,11 @@
     "sim:all": "pnpm sim:01-scan && pnpm sim:02-embed && pnpm sim:03-cluster && pnpm sim:04-plan && pnpm sim:05-write"
   },
   "dependencies": {
+    "@promethean/utils": "workspace:*",
```

Biome (2.1.2)

[error] The key @promethean/utils was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)